### ISSUE TITLE
Turso Driver Adapter: Including `_count` leads to error

### ISSUE THREAD

**Issue by piotrkulpinski** [prisma/prisma#23566]
### Bug description

When using Turso and attempting to generate an aggregated count, an error similar to the following is thrown

```
PrismaClientKnownRequestError:
Invalid `prisma.topic.findMany()` invocation:
Inconsistent column data: Conversion failed: expected a either an i64 or a f64 in column '_aggr_count_tools', found null
```
I think this error is thrown if the count for a specific relation equals 0.

This is the same as #23140 but that one was closed without solution.

### How to reproduce

With a Turso setup, run a query similar to the following:

```prisma
const topics = await prisma.topic.findMany({
  include: {
    _count: {
      select: {
        tools: true,
      },
    },
  },
})
```

### Prisma information

```prisma
model Tool {
  id             Int              @id @default(autoincrement())
  name           String
  slug           String           @unique
  updatedAt DateTime @updatedAt
  createdAt      DateTime         @default(now())
  topics          Topic[]
}

model Topic {
  id    Int             @id @default(autoincrement())
  name  String
  slug  String      @unique
  tools Tool[]
}
```

### Environment & setup

- OS: macOS
- Database: Turso (LibSQL)
- Bun 1.0.25

### Prisma Version

```
5.11.0
```

**Comment by SevInf** [comment]
Hi @piotrkulpinski.
Does the issue reproduce with an empty Turso database, or only when there is specific data in it? If later, could you provide a sample of that data?

**Comment by jkomyno** [comment]
Hi @piotrkulpinski, I wasn't able to reproduce this issue. Here's what I tried, using both Prisma versions `5.11.0` and `5.12.0-dev.45` (latest unstable version available). _Note: I've used Node.js 20.9.0, not Bun._

- `schema.prisma`
  ```prisma
  generator client {
    provider        = "prisma-client-js"
    previewFeatures = ["driverAdapters"]
  }
  
  datasource db {
    provider = "sqlite"
    url      = "file:./dev.db"
  }
  
  model Tool {
    id        Int      @id @default(autoincrement())
    name      String
    slug      String   @unique
    updatedAt DateTime @updatedAt
    createdAt DateTime @default(now())
    topics    Topic[]
  }
  
  model Topic {
    id    Int    @id @default(autoincrement())
    name  String
    slug  String @unique
    tools Tool[]
  }
  ```

- TypeScript snippet
  ```typescript
  import process from 'node:process'
  import { createClient } from '@libsql/client'
  import { PrismaLibSQL } from '@prisma/adapter-libsql'
  import { PrismaClient } from '@prisma/client'
  
  type Env = {
    TURSO_DATABASE_URL: string
    TURSO_AUTH_TOKEN: string
  }
  
  function getPrisma(env: Env) {
    const connectionString = `${env.TURSO_DATABASE_URL}`
    const token = `${env.TURSO_AUTH_TOKEN}`
  
    const turso = createClient({
      url: connectionString,
      authToken: token,
    })
    const adapter = new PrismaLibSQL(turso)
    return new PrismaClient({ adapter })
  }
  
  async function main() {
    const prisma = getPrisma(process.env as Env)
  
    await prisma.tool.deleteMany()
    await prisma.topic.deleteMany()
  
    await prisma.topic.create({
      data: {
        id: 1,
        name: 'Prisma',
        slug: 'prisma',
      }
    })
  
    const topics = await prisma.topic.findMany({
      include: {
        _count: {
          select: {
            tools: true,
          },
        },
      },
    })
  
    console.log('[topics]')
    console.dir(topics, { depth: null })
  }
  
  main()
  ```

  Output:

  ```sh
  [topics]
  [
    {
      id: 1,
      name: 'Prisma',
      slug: 'prisma',
      _count: { tools: 0 }
    }
  ]
  ```

Can you please provide your own reproduction to this issue, so we can evaluate it and hopefully fix it internally? Thanks.

**Comment by kwerdna19** [comment]
this is happening to me as well with prisma + Turso. It only seems to happens when 1 record has no relations and another does. Confirmed it doesn't happen when `adapter` is not passed to PrismaClient

here is my example:

schema.prisma
```
generator client {
  provider        = "prisma-client-js"
  previewFeatures = ["driverAdapters"]
}

datasource db {
  provider = "sqlite"
  url      = "file:./db.sqlite"
}

model User {
  id Int @id
  posts Post[]
}

model Post {
  id Int @id
  userId Int
  user User @relation(fields: [userId], references: [id], onDelete: Cascade)
}
```

test.ts
```
import { createClient } from "@libsql/client";
import { PrismaLibSQL } from "@prisma/adapter-libsql";
import { PrismaClient } from "@prisma/client";

const libSql = createClient({
	url: "file:./prisma/db.sqlite",
});

const adapter = new PrismaLibSQL(libSql);
const prisma = new PrismaClient({ adapter });

await prisma.user.deleteMany({});
await prisma.post.deleteMany({});

await prisma.user.create({
	data: {
		id: 1,
	},
});
// causes crash --
await prisma.user.create({
	data: {
		id: 2,
		posts: {
			create: {
				id: 123,
			},
		},
	},
});

// crashes - when 1 user has no posts and 1 does
await prisma.user.findMany({
	include: {
		_count: {
			select: {
				posts: true,
			},
		},
	},
});
```

crashes with error mentioned in post:

```
PrismaClientKnownRequestError: 
Invalid `prisma.user.findMany()` invocation:


Inconsistent column data: Conversion failed: expected a either an i64 or a f64 in column '_aggr_count_posts', found null
 code: "P2023"

      at new V (/Users/andrew/Desktop/zen-test/node_modules/@prisma/client/runtime/library.js:22:1414)
      at handleRequestError (/Users/andrew/Desktop/zen-test/node_modules/@prisma/client/runtime/library.js:122:6854)
      at handleAndLogRequestError (/Users/andrew/Desktop/zen-test/node_modules/@prisma/client/runtime/library.js:122:6183)
      at /Users/andrew/Desktop/zen-test/node_modules/@prisma/client/runtime/library.js:122:5891
```

**Comment by AbdulrhmanBagazi** [comment]
Any update on the issue am facing the same. did anyone find a solution?

**Comment by jkomyno** [comment]
Hi @piotrkulpinski, @AbdulrhmanBagazi and others. This issue was fixed via https://github.com/prisma/prisma-engines/pull/4893, and will be released in the next stable version of Prisma (5.15.0) early next week.

You can try it out right now with an (unstable) dev version of Prisma: `5.15.0-dev.50`. Thanks!

**Comment by Mandy9943** [comment]
I had same issue and now I saw the update and worked, good job!
